admin/testbrew.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script executed by Buildbot on the Mac mini build slave, this instantiates the
+OS X Virtual Machine to a snapshot with a clean homebrew install, then executes
+via SSH the brew update, install and test commands to verify a successful
+installation of a homebrew recipe.
+
+The host machine must have:
+        * A VMWare OS X VM available at a particular location
+
+The guest virtual machine must have:
+        * A snapshot with a particular name, homebrew-clean
+        * Homebrew installed and available
+"""
+
+import os
+import sys
+import urllib2
+
+from twisted.python.usage import Options, UsageError
+
+from subprocess import check_output, CalledProcessError
+
+from flocker.provision._install import run_with_fabric, task_test_homebrew
+
+
+YOSEMITE_VMX_PATH = (
+    '"'
+    "{home}/Documents/Virtual Machines.localized/"
+    "OS X 10.10.vmwarevm/OS X 10.10.vmx"
+    '"'
+).format(home=os.getenv("HOME"))
```

Comment:
The extra quotes around this value are a bit confusing.  They make this string not actually be the path it seems to be.  If they're necessary later, for example to be able to pass this to a shell command, they should be added later, not here.  I don't think they should be necessary anywhere, though (since you're using the non-shell form of `check_output`).  Does this default value actually work?  I would guess not... Perhaps it's not worth supplying a default value for this parameter.